src/view/shell/bottom-bar/BottomBarStyles.tsx

```diff
@@ -1,8 +1,11 @@
 import {StyleSheet} from 'react-native'
 import {colors} from 'lib/styles'
 
+export const BOTTOM_BAR_HEIGHT = 61
+
 export const styles = StyleSheet.create({
   bottomBar: {
+    height: BOTTOM_BAR_HEIGHT,
```

Comment:
This was its natural height in my browser, and since I wanted to use it as whitespace around the `Composer`, ideally we make it an explicit relationship so that updates here are reflected in the `Composer`.